nengo/networks/workingmemory.py

```diff
@@ -5,15 +5,13 @@
 
 
 def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
-                     difference_gain=1.0, reset_gain=3,
-                     mem_config=None, net=None):
+                     difference_gain=1.0, reset_gain=3, net=None):
     """Stores a given vector in memory, with input controlled by a gate."""
     if net is None:
         net = nengo.Network(label="Input Gated Memory")
 
-    if mem_config is None:
-        mem_config = nengo.Config(nengo.Connection)
-        mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
+    mem_config = nengo.Config(nengo.Connection)
+    mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
```

Comment:
Hmm....  on second thought, another option would be to get rid of ```difference_synapse``` altogether, and just set it to the same as ```recurrent_synapse```.  And maybe call it ```memory_synapse``` instead.

It sounds like the only people who would really want to play with making those parameters different are people mucking with the internals of the network, so they can just do that by changing the object model.  

That said, we don't have a hard and fast rule about where the cutoff of usefulness is, so it's probably fine either way.